parser_test.go

```diff
@@ -5,8 +5,86 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stripe/veneur/samplers"
+	"github.com/stripe/veneur/ssf"
 )
 
+func freshSSFMetric() *ssf.SSFSample {
```

Comment:
I found some bugs in some testing last night and I think we need a "valid metric" function akin to your valid trace function. A metric must have:
* Name
* Value
* Timestamp

If it doesn't we should discard it and increment a counter.

Lastly, I think we need to ensure a `SampleRate` of 1 if there's not one. I ran into a problem last night with that as well.